source/OptimaJet.DWKit.StarterApplication/Repositories/UserRepository.cs

```diff
@@ -6,17 +6,32 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Logging;
 using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Services;
 
 namespace OptimaJet.DWKit.StarterApplication.Repositories
 {
     public class UserRepository : DefaultEntityRepository<User>
     {
+        public IOrganizationContext OrganizationContext { get; }
+
         public UserRepository(
             ILoggerFactory loggerFactory,
             IJsonApiContext jsonApiContext,
+            IOrganizationContext organizationContext,
             IDbContextResolver contextResolver
             ) : base(loggerFactory, jsonApiContext, contextResolver)
         {
+            this.OrganizationContext = organizationContext;
+        }
+
+        public override IQueryable<User> Get()
+        {
+            if (OrganizationContext.InvalidOrganization) return base.Get().Take(0);
+            var users = base.Get().Include(u => u.OrganizationMemberships);
+            if (!OrganizationContext.HasOrganization) return users;
```

Comment:
this'll return all users in the database, yeah? 
don't we want to scope it to users in all the organizations the current user is in?

Reply:
The current user can be in multiple organizations.  The organization switcher will select one organization.  Will it be able to select multiple organizations?

Where in the UI will be be viewing a list of users anyway?  Just curious.

Reply:
the current requirements are 'one org' or 'all of the orgs a user belongs to'.

that'll be in @giancorzo 's PR: https://github.com/sillsdev/appbuilder-portal/pull/31